packages/types/src/types/requests/datatrak-web-server/SubmitSurveyRequest.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+import { UserAccount, Survey, Entity } from '../../models';
+import { SurveyScreenComponent } from './SurveyRequest';
+
+export type AutocompleteAnswer = {
+  isNew?: boolean;
+  optionSetId: string;
+  value: string;
+  label: string;
+};
+
+type Answer = string | number | boolean | null | undefined | AutocompleteAnswer;
+
+export type Answers = Record<string, Answer>;
+
+type Question = SurveyScreenComponent;
```

Comment:
See my other comment about `Question.questionId`

I think I'd generally prefer to keep the type name relative to the related db table, even if that means we have something that looks a little weird like `questions: SurveyScreenComponents[]`

Or possibly we could use a mixed name like `ScreenQuestion` just to make it a bit more clear that this isn't directly related to the db question type

Reply:
Yes fair point. From looking through the types definitions there is a convention to use SurveyScreenComponent as the type name so I've just updated it here to be that. I will think about adding a new naming convention in a later PR as I don't want this PR to drag on too much more.